Meadowlark-js/services/meadowlark-fastify/src/Service.ts

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+import { randomUUID } from 'node:crypto';
```

Comment:
Totally forgot node has its own crypto package. We should switch from jsSha to this in core.

Reply:
That's a format I've not seen before: `node:crypt` with a colon. Removing that library makes sense: one fewer dependency to worry about, and slightly slimmer file distribution. I'll create a ticket.

Reply:
Yeah, the "node:" prefix is relatively new. The idea is to avoid npm typosquatting issues for built-in node libraries.